internal/service/container/restart.go

```diff
@@ -5,28 +5,36 @@ package container
 
 import (
 	"context"
-	"time"
 
+	"github.com/containerd/nerdctl/v2/pkg/api/types"
 	"github.com/runfinch/finch-daemon/pkg/errdefs"
 )
 
-func (s *service) Restart(ctx context.Context, cid string, timeout time.Duration) error {
-	con, err := s.getContainer(ctx, cid)
+func (s *service) Restart(ctx context.Context, containers []string, options types.ContainerRestartOptions) error {
```

Comment:
Why do we need to change from taking `cid string` to `containers []string`?

Reply:
Nerdctl currentl accepts container id's in [[]string format](https://github.com/containerd/nerdctl/blob/89003a58835a7907687ffb46d3d133349cfc7ce9/pkg/cmd/container/start.go#L31C60-L31C64).
I think making our functions take cid in input was the right approach, we can send the cid in the format that nerdctl expects when we make that call. 

Starting multiple containers is not a feature that docker supports at the moment - https://docs.docker.com/reference/api/engine/version/v1.43/#tag/Container/operation/ContainerStart

I will update this.

Reply:
Sounds good, thanks